tutorials/workshop_flowsheet.py

```diff
@@ -451,12 +451,12 @@ class Category:
                 "display_units": "{USD_2018}/(m**3/hr)",
                 "category": Category.cost,
             },
-            "load_factor": {
-                "display_name": "Load factor",
-                "scale_factor": 100,
-                "display_units": "%",
-                "category": Category.cost,
-            },
+            # "load_factor": {
```

Comment:
It's name got changed to "utilitization_factor", we can probably just update it to that rather than removing it